src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int FIGURE_COUNT = 5;
+    private static final int MAX_SIDE_LENGTH = 5;
+
+    private static final Random RANDOM = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
цей клас не потрібно робити константою